index.md

```diff
@@ -383,10 +383,5 @@ These are the installation instructions for the tools used
 during the workshop.
 {% endcomment %}
 
-{% if site.carpentry == "swc" %}
-{% include swc/setup.html %}
-{% elsif site.carpentry == "dc" %}
-{% include dc/setup.html %}
-{% elsif site.carpentry == "lc" %}
-{% include lc/setup.html %}
-{% endif %}
+In order to prepare for the workshpo follow our
```

Comment:
It is easy to miss this when it's at the bottom. I would actually put it in the first sentence of this block (To participate in a Data Carpentry workshop, you will need access to the software described below. In addition, you will need an up-to-date web browser.)